py-polars/src/file.rs

```diff
@@ -210,17 +200,12 @@ pub fn get_either_file(py_f: PyObject, truncate: bool) -> PyResult<EitherRustPyt
             let file_path = std::path::Path::new(&s);
             let file_path = resolve_homedir(file_path);
             let f = if truncate {
-                BufReader::new(File::create(file_path)?)
+                File::create(file_path)?
```

Comment:
This can have performance implications.

Reply:
Huh, really? We are still creating a `BufReader` on line 207. Does allocating the `File` object to a variable cost us performance here?

Reply:
Oh, sorry, I didn't see that. I thought the bufreader was removed.